src/js/components/agencyV2/AgencyPage.jsx

```diff
@@ -106,52 +106,40 @@ export const AgencyProfileV2 = ({
     };
 
     return (
-        <div className="usa-da-agency-page-v2">
-            <MetaTags {...agencyPageMetaTags} />
-            <Header />
-            <PageHeader
-                overLine="Agency Profile"
-                title={name}
-                stickyBreakPoint={getStickyBreakPointForSidebar()}
-                fyProps={{
-                    selectedFy,
-                    latestFy,
-                    handleFyChange: (fy) => setSelectedFy({ fy })
-                }}
-                shareProps={{
-                    url: getBaseUrl(slug),
-                    onShareOptionClick: handleShare
-                }}
-                downloadProps={{
-                    downloadInFlight: false,
-                    handleClick: () => {}
-                }}>
-                <main id="main-content" className="main-content usda__flex-row">
-                    <div className="sidebar usda__flex-col">
-                        <Sidebar
-                            pageName="agency-v2"
-                            fixedStickyBreakpoint={scrollPositionOfSiteHeader}
-                            isGoingToBeSticky
-                            active={activeSection}
-                            jumpToSection={jumpToSection}
-                            detectActiveSection={setActiveSection}
-                            sections={sections.map((section) => ({
-                                section: section.name,
-                                label: section.display,
-                                overLine: section.overLine
-                            }))} />
-                    </div>
-                    <div className="body usda__flex-col">
-                        {isError ? <ErrorMessage /> : sections.map((section) => (
-                            <AgencySection key={section.name} section={section} isLoading={isLoading}>
-                                {section.component || <ComingSoon />}
-                            </AgencySection>
-                        ))}
-                    </div>
-                </main>
-                <Footer />
-            </PageHeader>
-        </div>
+        <PageWrapper
```

Comment:
Missing padding in the header on this page too

<img width="1594" alt="Screen Shot 2021-05-07 at 4 59 20 PM" src="https://user-images.githubusercontent.com/7108785/117507839-9ca3e600-af55-11eb-9026-7169e5a26559.png">